store/spot.ts

```diff
@@ -271,6 +271,30 @@ export const actions = actionTree(
       commit('setMarketsSummary', actualMarketsSummary as UiSpotMarketSummary[])
     },
 
+    async getMarkets({ commit }) {
```

Comment:
the codes here are diplicated in the `init()` function.

we can either:
1. refactor init function to call `this.$accessor.spot.fetchMarkets()`
2. or remove the `getMarkets` function and use `this.$accessor.spot.init()` on the swap page mounted hook